apps/studio/components/interfaces/Integrations/Vault/Secrets/SecretRow.tsx

```diff
@@ -95,31 +104,24 @@ const SecretRow = ({ secret, onSelectEdit, onSelectRemove }: SecretRowProps) =>
           {secret.updated_at === secret.created_at ? 'Added' : 'Updated'} on{' '}
           {dayjs(secret.updated_at).format('MMM D, YYYY')}
         </p>
-        <DropdownMenu>
-          <DropdownMenuTrigger asChild>
-            <Button type="text" className="px-1" icon={<MoreVertical />} />
-          </DropdownMenuTrigger>
-          <DropdownMenuContent side="bottom" align="end" className="w-32">
-            <Tooltip>
-              <TooltipTrigger asChild>
-                <DropdownMenuItem
-                  className="space-x-2"
+
+        <Tooltip>
```

Comment:
same thing RE consistency - we aren't currently following a pattern of disabling the dropdown menu button here

i'd stick to keeping the dropdown menu clickable, but disabling the menu items with DropdownMenuItemTooltip instead ([example](https://github.com/supabase/supabase/blob/master/apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyRow.tsx#L152))

this is probably worth relooking as a pattern across the dashboard, but i'd say we do that separately